<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="yuhualaideMacBook-Pro.local" ignored="0" name="Default test" tests="1" failures="1" timestamp="23 五月 2018 04:01:00 GMT" time="26.215" errors="0">
  <testcase name="Test" time="21.426" classname="com.liangyihui.intal.TestLogin">
    <failure type="java.lang.AssertionError" message="两个值不一样 expected [【JOO乳腺癌文献荟萃55】与NSCLC患者相比，乳腺癌患者需筛查脑转移吗？|早期乳腺癌患者开始曲妥珠...] but found [【JOO乳腺癌文献荟萃55】与NSCLC患者相比，乳腺癌患者需筛查脑转移吗？|早期乳腺癌患者开始曲妥珠单抗时...]">
      <![CDATA[java.lang.AssertionError: 两个值不一样 expected [【JOO乳腺癌文献荟萃55】与NSCLC患者相比，乳腺癌患者需筛查脑转移吗？|早期乳腺癌患者开始曲妥珠...] but found [【JOO乳腺癌文献荟萃55】与NSCLC患者相比，乳腺癌患者需筛查脑转移吗？|早期乳腺癌患者开始曲妥珠单抗时...]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at com.liangyihui.intal.TestLogin.Test(TestLogin.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- Test -->
</testsuite> <!-- Default test -->
